dts/arm/st/h5/stm32h563Xi.dtsi

```diff
@@ -8,8 +8,22 @@
 #include <st/h5/stm32h563.dtsi>
 
 / {
-	sram0: memory@20000000 {
-		reg = <0x20000000 DT_SIZE_K(640)>;
+	sram1: memory@20000000 {
+		compatible = "zephyr,memory-region", "mmio-sram";
+		reg = <0x20000000 DT_SIZE_K(256)>;
+		zephyr,memory-region = "SRAM1";
+	};
```

Comment:
I guess aim of renaming sram0 to sram1 is to be closer to ref man nomenclature.
I need to investigate potential impact of having "main" ram with a different name from sram0

Reply:
ok

Reply:
Looks fine, let's keep it that way.